lib/intrinsic-jsx.d.ts

```diff
@@ -22,6 +22,7 @@ declare global {
       torus: FN.TorusProps
       custom: FN.CustomProps
       union: FN.UnionProps
+      rotate: FN.RotateProps
```

Comment:
```suggestion
      rotate: { args: [number, number, number]; children: any }
```

Reply:
Why do we need to change the prop to args rather than angle?
Rotate only accepts angle as a prop.

Reply:
most people don't consider angle to be a point3 object, jscad calls it `out` which is even worse. So basically `args` feels like the most neutral answer, since we're just passing it to jscad anyway. Most people will use `<Rotate />` not `<rotate />` so it's not a huge issue for end users.

I'm ok with either solution in this case, but we still have to change this from `RotateProps` which is different because `angle` can be a `Point3` which is `[number,number,number]` or `{x,y,z}`

```suggestion
      rotate: { angle: [number, number, number]; children: any
```